site2/website/versioned_docs/version-2.1.1-incubating/developing-binary-protocol.md

```diff
@@ -68,7 +68,7 @@ If you want to use broker entry metadata for **consumers**:
 
 1. Use the client protocol version [18 or later](https://github.com/apache/pulsar/blob/ca37e67211feda4f7e0984e6414e707f1c1dfd07/pulsar-common/src/main/proto/PulsarApi.proto#L259).
    
-2. Configure the [`brokerEntryMetadataInterceptors`](reference-configuration.md#broker) parameter and set the [`enableExposingBrokerEntryMetadataToClient`](reference-configuration.md#broker) parameter to `true` in the `broker.conf` file.
```

Comment:
The brokerEntryMetadataInterceptors should not appear before V2.10.0. It's the same with https://github.com/apache/pulsar/pull/17004

Reply:
`brokerEntryMetadataInterceptors` was introduced since 2.8.0, but `enableExposingBrokerEntryMetadataToClient` was introduced since 2.10.0.